kytos/core/controller.py

```diff
@@ -703,3 +709,27 @@ def unload_napps(self):
         # items from it.
         for (username, napp_name) in list(self.napps.keys()):  # noqa
             self.unload_napp(username, napp_name)
+
+    def reload_napp(self, username, napp_name):
+        """Reload a NApp."""
+        try:
+            napp_module = self._import_napp(username, napp_name)
+        except ModuleNotFoundError as err:
+            self.log.error("Error loading NApp '%s/%s': %s",
+                           username, napp_name, err)
+            return
+
+        napp_module = reload_module(napp_module)
+        self.napps[(username, napp_name)] = napp_module.Main(controller=self)
```

Comment:
There's no indication that it worked successfully. Please add an INFO log line to Kytos console to tell that a napp was reloaded.